facade/disabledNodeFacade.go

```diff
@@ -0,0 +1,245 @@
+package facade
+
+import (
+	"errors"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/api/address"
+	"github.com/ElrondNetwork/elrond-go/api/block"
+	"github.com/ElrondNetwork/elrond-go/api/hardfork"
+	"github.com/ElrondNetwork/elrond-go/api/node"
+	transactionApi "github.com/ElrondNetwork/elrond-go/api/transaction"
+	"github.com/ElrondNetwork/elrond-go/api/validator"
+	"github.com/ElrondNetwork/elrond-go/api/vmValues"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/statistics"
+	"github.com/ElrondNetwork/elrond-go/core/vmcommon"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/data/vm"
+	"github.com/ElrondNetwork/elrond-go/debug"
+	"github.com/ElrondNetwork/elrond-go/heartbeat/data"
+	"github.com/ElrondNetwork/elrond-go/node/external"
+	"github.com/ElrondNetwork/elrond-go/ntp"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+var _ = address.FacadeHandler(&disabledNodeFacade{})
+var _ = hardfork.FacadeHandler(&disabledNodeFacade{})
+var _ = node.FacadeHandler(&disabledNodeFacade{})
+var _ = transactionApi.FacadeHandler(&disabledNodeFacade{})
+var _ = validator.FacadeHandler(&disabledNodeFacade{})
+var _ = vmValues.FacadeHandler(&disabledNodeFacade{})
+
+var errNodeStarting = errors.New("Node is starting")
+var emptStr string = ""
+
+// nodeFacade represents a facade for grouping the functionality for the node
+type disabledNodeFacade struct {
+}
+
+// NewDisableNodeFacade creates a new Facade with a NodeWrapper
+func NewDisableNodeFacade() (*disabledNodeFacade, error) {
```

Comment:
NewDisabledNodeFacade*
also, you can drop the second param (`error`) because there is no way that this constructor can fail

Reply:
done